Userland/Applications/FileManager/FileUtils.cpp

```diff
@@ -67,4 +69,104 @@ void delete_paths(const Vector<String>& paths, bool should_confirm, GUI::Window*
     }
 }
 
+void create_trash_bin()
+{
+    auto trash_bin_path = Core::StandardPaths::trash_directory();
+    auto trash_bin_files = String::formatted("{}/files", trash_bin_path);
+    auto trash_bin_info = String::formatted("{}/info", trash_bin_path);
+
+    if (!Core::File::is_directory(trash_bin_path)) {
+        if (mkdir(trash_bin_path.characters(), 0755) < 0) {
+            auto saved_errno = errno;
+            warnln("Failed to create directory {}: {}", trash_bin_path, strerror(saved_errno));
+        }
+    }
+
+    if (!Core::File::is_directory(trash_bin_files)) {
+        if (mkdir(trash_bin_files.characters(), 0755) < 0) {
+            auto saved_errno = errno;
+            warnln("Failed to create directory {}: {}", trash_bin_files, strerror(saved_errno));
+        }
+    }
+
+    if (!Core::File::is_directory(trash_bin_info)) {
+        if (mkdir(trash_bin_info.characters(), 0755) < 0) {
+            auto saved_errno = errno;
+            warnln("Failed to create directory {}: {}", trash_bin_info, strerror(saved_errno));
+        }
+    }
+}
+
+[[maybe_unused]] bool move_to_trash(String const& path)
+{
+    [[maybe_unused]] auto file_info = "[Trash Info]\nPath={}\nDeletionDate={}\n";
```

Comment:
This format string is declared here with `[[maybe_unused]]`. It is only ever used once. There's no reason to define it as a separate variable.

More importantly, the INI library should be used for writing INI data, instead of using format strings.

Reply:
Hmm do you mean something like this:

```C++
auto file = file_or_error.release_value();
    file->write("");
    file->close();

    auto trash_info = Core::ConfigFile::open(trash_info_path);

    trash_info->write_entry("Trash Info", "Path", path);
    trash_info->write_entry("Trash Info", "DeletionDate", deletion_date);
    trash_info->sync();
```

Reply:
> Hmm do you mean something like this:

Something like that, yes. Although I'm not sure why the `file->write()` would be required.